pom.xml

```diff
@@ -149,6 +150,7 @@
     <module>shell</module>
     <module>examples</module>
     <module>integration-tests</module>
+    <module>integration</module>
```

Comment:
so the `integration/pom.xml` will be picked up by maven by default,
and then when `developer` profile is selected, `integration/{yarn,mesos}/pom.xml` can inherit correct property values like `yarn.version`